packages/nextjs/app/builders/0xd1B41bE30F980315b8A6b754754aAa299C7abea2/page.tsx

```diff
@@ -0,0 +1,119 @@
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const profileUrl = "https://pbs.twimg.com/media/Ey8oK6gXMAUKrEX.jpg";
+
+const RolandTheFrank: NextPage = () => {
+  return (
+    <div className="w-96 px-6 py-6  text-center bg-gray-800 rounded-lg lg:mt-0 xl:px-10">
```

Comment:
It would be great if you could center your personal page box both horizontally and vertically on the page.